builder/builder.go

```diff
@@ -19,6 +19,8 @@ var (
 	errLimitValueLength          = errors.New(`[builder] the value of "_limit" must contain one or two uint elements`)
 	errHavingValueType           = errors.New(`[builder] the value of "_having" must be of map[string]interface{}`)
 	errHavingUnsupportedOperator = errors.New(`[builder] "_having" contains unsupported operator`)
+	errLockValueType             = errors.New(`[builder] the value of "_lock" must be of string type`)
+	errNotAllowedLockType        = errors.New(`[builder] the value of "_lock" is not allowed`)
```

Comment:
`_lockMode` as we actually use this word.

Reply:
ok, I will be change the error content, thank u.

Reply:
> `_lockMode` as we actually use this word.

what about this?

Reply:
Sounds good to me, ping @caibirdme for further review.

Reply:
> Sounds good to me, ping @caibirdme for further review.

All right, thank u.

Reply:
@caibirdme Please review the code, thank u.